src/other/sample.jl

```diff
@@ -0,0 +1,29 @@
+import StatsBase: sample
+
+function sample(df::AbstractDataFrame; replace::Bool=true, ordered::Bool=false)
+    sample(df, 1, replace=replace, ordered=ordered)
+end
+
+"""
+    sample(df[, N])
+
+Returns a (random) sample of N rows from a DataFrame
```

Comment:
Backquotes around `N`, which should actually be lower-case `n` for consistency with StatsBase.

Also, at https://github.com/JuliaStats/StatsBase.jl/pull/172, the wording is "Draw a random sample of elements from an array `a`. So probably better be consistent and say "Draw a random sample of `N` rows from a data frame `df` and return the result as a data frame."

Finally, you need to document the keyword arguments:
```
# Arguments

- `replace::Bool=true`: Should sampling be performed with replacement?
 • `ordered::Bool=false`: Should an ordered sample be taken?
```